test/python/algorithms/optimizers/test_optimizer_aqgd.py

```diff
@@ -70,7 +71,12 @@ def test_list(self):
         )
 
         aqgd = AQGD(maxiter=[1000, 1000, 1000], eta=[1.0, 0.5, 0.3], momentum=[0.0, 0.5, 0.75])
-        vqe = VQE(ansatz=RealAmplitudes(), optimizer=aqgd, quantum_instance=q_instance)
+        vqe = VQE(
+            ansatz=RealAmplitudes(),
+            optimizer=aqgd,
+            quantum_instance=q_instance,
+            include_custom=True,
```

Comment:
is this because the previous behaviour changed?

Reply:
Ah, right. It is true that the algorithm used here has changed, so it is more correct to use the Statevector simulator for this simulation.